pkg/tracer/tracer.go

```diff
@@ -905,6 +906,20 @@ func (m *FlowFetcher) LookupAndDeleteMap(met *metrics.Metrics) map[ebpf.BpfFlowI
 	return flows
 }
 
+func (m *FlowFetcher) increaseEnrichmentStats(met *metrics.Metrics, flow *model.BpfFlowContent) {
+	if flow.AdditionalMetrics != nil {
+		met.FlowEnrichmentCounter.Increase(
+			flow.AdditionalMetrics.DnsRecord.Id != 0,
+			flow.AdditionalMetrics.FlowRtt != 0,
+			flow.AdditionalMetrics.PktDrops.Packets != 0,
+			flow.AdditionalMetrics.NetworkEventsIdx != 0,
+			flow.AdditionalMetrics.TranslatedFlow.ZoneId != 0,
```

Comment:
zoneid of 0 is valid

Reply:
hmm :/
I wanted to avoid something more costly, so I guess I need to check for empty IPs ...